packages/squiggle-lang/src/rescript/utility/Sparklines.res

```diff
@@ -0,0 +1,40 @@
+// Port of Sindre Sorhus' Sparkly to Rescript
```

Comment:
I imagine a lot of this code was taken very directly, but think that ~20min of refactoring this would make it much nicer, and be worth it